data/json/items/tool_armor.json

```
@@ -2778,8 +2812,8 @@
     "color": "light_gray",
     "material_thickness": 3,
     "use_action": [ "SOLARPACK" ],
-    "flags": [ "FRAGILE", "OUTER", "ONLY_ONE", "SOLARPACK" ],
-    "armor": [ { "encumbrance": 12, "coverage": 40, "covers": [ "torso" ] } ]
+    "flags": [ "BELTED", "FRAGILE", "OUTER", "ONLY_ONE", "SOLARPACK" ],
```

```suggestion
    "flags": [ "BELTED", "FRAGILE", "ONLY_ONE", "SOLARPACK" ],
```

Should probably be just belted not double layered